packages/extension/gulpfile.ts

```diff
@@ -13,7 +13,7 @@ async function cypressIcons () {
 }
 
 const clean = (done) => {
-  rimraf('dist', done)
+  rimraf('dist').then(() => done())
 }
```

Comment:
We could switch this to be even cleaner by making this an async function.

```javascript
const clean = async () => {
  await rimraf('dist')
}
```

Reply:
I've found that in Gulp, returning a Promise is enough to make a task asynchronous. Therefore, returning the rimraf execution result directly is sufficient.